tests/zenko_tests/node_tests/cloudserver/keyFormatVersion/tests/versionedBucket.js

```diff
@@ -0,0 +1,147 @@
+const assert = require('assert');
+const async = require('async');
+
+const s3 = require('../../../s3SDK').scalityS3Client;
+
+const bucket = 'key-format-versioned-bucket';
+
+const versionIds = {
+    'first-key': [],
+    'second-key': [],
+};
+
+function putObjectVersions(objName, cb) {
+    async.timesSeries(3, (n, next) => {
+        s3.putObject({ Bucket: bucket, Key: objName }, (err, res) => {
+            assert.ifError(err);
+            versionIds[objName].push(res.VersionId);
+            return next();
+        });
+    }, cb);
+}
+
+function deleteObjectVersions(objName, cb) {
+    async.timesSeries(3, (n, next) => {
+        s3.deleteObject({ Bucket: bucket, Key: objName, VersionId: versionIds[objName][n] }, next);
+    }, cb);
+}
+
+describe('Cloudserver : keyFormatVersion : versioned bucket', () => {
+    before(done => {
+        async.series([
+            next => s3.createBucket({ Bucket: bucket }, next),
+            next => {
+                const params = {
+                    Bucket: bucket,
+                    VersioningConfiguration: {
+                        MFADelete: 'Disabled',
+                        Status: 'Enabled',
+                    },
+                };
+                return s3.putBucketVersioning(params, next);
+            },
+            next => putObjectVersions('first-key', next),
+            next => putObjectVersions('second-key', next),
+        ], err => {
+            assert.ifError(err);
+            done();
+        });
+    });
+
+    after(done => {
+        async.series([
+            next => deleteObjectVersions('first-key', next),
+            next => deleteObjectVersions('second-key', next),
+            next => s3.deleteBucket({ Bucket: bucket }, next),
+        ], err => {
+            assert.ifError(err);
+            done();
+        });
+    });
+
+    it('Should return metadata of last version', done => {
+        const params = {
+            Bucket: bucket,
+            Key: 'first-key',
+        };
+        s3.getObject(params, (err, data) => {
+            assert.ifError(err);
+            assert.strictEqual(data.VersionId, versionIds['first-key'][2]);
```

Comment:
should verify returned object key

Reply:
`getObject` does not return the object key